### DISCUSSION TITLE
Stack trace missing in allure report

### DISCUSSION THREAD

**Discussion by shemeshsapir** [allure-framework/allure2#2713] [Questions & Support] [answered]
Hi, I'm using the following packages in my project.:
"allure-codeceptjs": "^2.15.1",
"allure-commandline": "^2.13.0",

and I expected to see stack trace of failure as I had in "allure-legacy" package 
and it is missing now.
It's just show "Empty status details".

Is it related to the version I use? Is it fixes in newer version?

![image](https://github.com/user-attachments/assets/2fa868da-e5b1-47bd-9fcc-d0b17216c6eb)

**Comment by baev** [comment, marked as answer]
@shemeshsapir please try [allure-codeceptjs@3.0.0-beta.11](https://www.npmjs.com/package/allure-codeceptjs/v/3.0.0-beta.11)
